docs/rules/Moq1002.md

```
@@ -1,10 +1,10 @@
 # Moq1002: Parameters provided into mock do not match any existing constructors
 
-| Item | Value |
-| --- | --- |
-| Enabled | True |
+| Item     | Value   |
+| -------- | ------- |
+| Enabled  | True    |
 | Severity | Warning |
-| CodeFix | False |
+| CodeFix  | False   |
```

Markdownlint

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)